### PR TITLE
PMM-7820 Add toggle to enable/disable DBaaS

### PR DESCRIPTION
**Pull request by jprukner**
We are now able to enable/disable dbaas using toggle. We also added ENABLE_DBAAS environment variable that will replace PERCONA_TEST_DBAAS eventually.

I don't know how to generate the site so I was not able to test it. @PaulJacobs-percona please take a look, is there any tutorial on how to generate the site so I can view it and click through my changes?

### CODE REVIEW

**Comment by jprukner** [comment]
```
Traceback (most recent call last):
  File "/opt/buildhome/python3.7/lib/python3.7/site-packages/mkdocs_macros/plugin.py", line 443, in render
    return md_template.render(**page_variables)
  File "/opt/buildhome/python3.7/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/opt/buildhome/python3.7/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/opt/buildhome/python3.7/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 13, in top-level template code
jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'toggle'
```
ok, so there is a preview deploy, neat! :laughing: 
I thought I'd put icon of toggle in there but it seems to not work @PaulJacobs-percona Maybe I need to escape it so it takes `toggle-off` as the thing I want, not `toggle`.

File: docs/setting-up/server/dbaas.md [hunk 1]
```diff
@@ -40,13 +40,13 @@ ln -s /usr/local/sbin/minikube /usr/sbin/minikube
...
 
-## Start PMM server with DBaaS activated
+## Start PMM server and activate a DBaaS feature
 
 > - To start a fully-working 3 node XtraDB cluster, consisting of sets of 3x HAProxy, 3x PXC and 6x PMM Client containers, you will need at least 9 vCPU available for minikube. (1x vCPU for HAProxy and PXC and 0.5vCPU for each pmm-client containers).
 >
 > - DBaaS does not depend on PMM Client.
 >
-> - Setting the environment variable `PERCONA_TEST_DBAAS=1` enables DBaaS functionality.
+> - You can pass the environment variable `--env ENABLE_DBAAS=1` to force the DBaaS feature when starting up pmm-server container. **You can omit the variable and enable the feature later using PMM UI**, please follow the link in step 3. bellow.
```
Comment (PaulJacobs-percona) [inline docs/setting-up/server/dbaas.md:49]: /bellow/below

File: docs/setting-up/server/dbaas.md [hunk 2]
```diff
@@ -66,6 +66,10 @@ alias kubectl='minikube kubectl --'
     docker exec -t pmm-server bash -c 'ln -s /srv/grafana /usr/share/grafana/data; chown -R grafana:grafana /usr/share/grafana/data; grafana-cli --homepath /usr/share/grafana admin reset-admin-password <RANDOM_PASS_GOES_IN_HERE>'
     ```
 
+3. ***IMPORTANT***: *Please follow instructions on* [**How to activate the *DBaaS* feature in Advanced Settings of PMM**](../../using/platform/dbaas.md#activate-a-dbaas-feature).
```
[the remark this answers is not in the document]
Comment (PaulJacobs-percona) [reply docs/setting-up/server/dbaas.md:69]: I expect the `ENABLE_DBAAS` envvar will be deprecated eventually so you are right to prefer explaining the in-app way of activating it.

File: docs/using/platform/dbaas.md [hunk 3]
```diff
@@ -6,12 +6,14 @@
 
 The DBaaS dashboard is where you add, remove, and operate on Kubernetes and database clusters.
 
-> You must run PMM Server with a DBaaS feature flag to activate the features described here.
+## Activate a DBaaS feature
+
+**To activate the *DBaaS* feature**, select {{ icon.bars }} *PMM* --> *PMM Settings* --> *Advanced settings*.
+Then turn on the DBaaS feature by clicking on {{ icon.toggle-off }} toggle in the *Technical preview features* section of the page.
```
Comment (PaulJacobs-percona) [inline docs/using/platform/dbaas.md:12]: The icons are defined as variables in `variables.yml`, and as far as I recall can't have dashes in the names. For example: `  times: '<i class="uil uil-times"></i>'` is then used as  `icon.times`.
The unicons icons are listed at see https://iconscout.com/unicons/explore/line. You (or I) will need to add them to make the build work.

**Comment by PaulJacobs-percona** [comment]
> ```
> Traceback (most recent call last):
>   File "/opt/buildhome/python3.7/lib/python3.7/site-packages/mkdocs_macros/plugin.py", line 443, in render
>     return md_template.render(**page_variables)
>   File "/opt/buildhome/python3.7/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
>     self.environment.handle_exception()
>   File "/opt/buildhome/python3.7/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
>     reraise(*rewrite_traceback_stack(source=source))
>   File "/opt/buildhome/python3.7/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
>     raise value.with_traceback(tb)
>   File "<template>", line 13, in top-level template code
> jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'toggle'
> ```
> 
> ok, so there is a preview deploy, neat! 😆
> I thought I'd put icon of toggle in there but it seems to not work @PaulJacobs-percona Maybe I need to escape it so it takes `toggle-off` as the thing I want, not `toggle`.

See my comment elsewhere about how icons are used. (Look in `variables.yml` and see if you can work it out. If not, I'll add them.)

File: docs/setting-up/server/dbaas.md (hunk 1 above)
Comment (jprukner) [reply docs/setting-up/server/dbaas.md]: fixed

File: docs/using/platform/dbaas.md (hunk 3 above)
Comment (jprukner) [reply docs/using/platform/dbaas.md]: I added the toggle-off icon. It seems to work well.

**Comment by jprukner** [comment]
Both FE ticket, PMM-7641, and BE ticket, PMM-7820, are In Docs stage now so we can merge this if everything is ok @tiagomotasantos @PaulJacobs-percona

File: docs/setting-up/server/dbaas.md [hunk 4]
```diff
@@ -66,6 +66,10 @@ alias kubectl='minikube kubectl --'
...
+
+    You need to enable the feature using PMM UI if you omited `--env ENABLE_DBAAS=1` when starting up the container.
```
Comment (tiagomotasantos) [inline docs/setting-up/server/dbaas.md:71]:
```suggestion
    You need to enable the feature using PMM UI if you omitted `--env ENABLE_DBAAS=1` when starting up the container.
```

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]
Great job @jprukner!

**Review by tiagomotasantos — APPROVED** [review]

**Comment by PaulJacobs-percona** [comment]
> We are now able to enable/disable dbaas using toggle. We also added ENABLE_DBAAS environment variable that will replace PERCONA_TEST_DBAAS eventually.
> 
> I don't know how to generate the site so I was not able to test it. @PaulJacobs-percona please take a look, is there any tutorial on how to generate the site so I can view it and click through my changes?

README of this repo should be helpful for generating a local preview.

**Review by PaulJacobs-percona — APPROVED** [review]

**Pull request merged**